softwerkskammer/test/middleware/accessrights_test.js

```diff
@@ -27,6 +27,14 @@ function superuser() {
   return standardMember({id: 'superuserID'});
 }
 
+function goupWithContactTheOrganizersTurnedOff() {
```

Comment:
Bitte lieber "const" bei den Aufrufern (s.o.)
Über den Tipfehler schweige ich lieber... ;)

Reply:
Der Tippfehler ist gekauft, also behoben ;-).
Aber direkt über `goupWithContactTheOrganizersTurnedOff` wird mit demselben Pattern auch per Methode Testdaten erstellt (z. B. `standardMember` oder auch `superuser`). War eventuell auch ein Grund warum wir das so gemacht haben. Welche Variante ist jetzt die bevorzugte?
In `mailsenderService_test` habe ich das jetzt schon umgestellt, wobei ich es teilweise vorher schöner fand (z. B. bei `memberWithMailAdress` (typo würde ich auch noch beheben) wo man einfach die Adresse übergeben konnte und gut wars), jetzt wo ich das oben gesehen habe, würde ich doch nochmal das Fass auf machen und darüber diskutieren ;-).